ha/util/conf_store.py

```diff
@@ -28,17 +28,23 @@
 from ha.k8s_setup.const import _DELIM, GconfKeys
 from cortx.utils.log import Log
 from ha.core.error import ClusterCardinalityError
+from ha.const import CLUSTER_GLOBAL_INDEX
+from cortx.utils.const import CLUSTER_CONF
 
 
 class ConftStoreSearch:
     """
     Wrapper class for using the confstore search APIs
     """
+    _cluster_conf_loaded = False
 
     def __init__(self, conf_store_req: bool = True):
         """Init method"""
         if conf_store_req:
             self._confstore = ConfigManager.get_confstore()
+        if not ConftStoreSearch._cluster_conf_loaded:
```

Comment:
Please create the new ticket as discussed. In the DoD along with other changes, also add a point that the relevant code added in this PR should be removed.

Reply:
sure

Reply:
https://jts.seagate.com/browse/CORTX-29597 is created